src/indexv3.js

```diff
@@ -0,0 +1,58 @@
+var interviewApp = {};
+(function(){
+    const container = document.getElementById(`app`);
+    let module = null;
+    
+    
+    const checkIfLogged = function(){
+        const isLogged = function() {
+            return !!window.localStorage.getItem('userData');
+        }
+        //console.log(isLogged())
+        if(!isLogged()){
+            return 'login'
+        }
+        return ''
+    }
+
+
+    interviewApp.navigate = function(page){
+        //console.log(page)
+        if (typeof page !== 'string') {
+            page = 'login';
+        }
+        if(module){
+            module.destroy();
+        }
+        if(checkIfLogged() != ''){
+            page = checkIfLogged();
+        }
+        module = interviewApp[page];
+
+
+        
+            if(localStorage.getItem("evaluation") !== null){
+                q = JSON.parse(localStorage.getItem("evaluation")).map(function(element){
+                  return {
+                    nume:element.inputCandidate[0][0],
+                    technologie:element.textArea[2].input,
+                    nivel: element.radioBox,
+                    buttonIMG: "/assets/button.jpg"
```

Comment:
Can you explain what this is doing? At first glance, it looks as if it belongs in the presentation (i.e. CSS) layer instead.